config.ts

```diff
@@ -0,0 +1,27 @@
+
+const missing_vars = [];
+const env_var_defaults = {
+    PORT:                   8001,
```

Comment:
generally we've erred toward not setting any defaults.
thoughts? were you modeling this after another repo?